MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionPresenter.cpp

```diff
@@ -1084,7 +1099,31 @@ void EnggDiffractionPresenter::doFocusing(const EnggDiffCalibSettings &cs,
     alg->execute();
 
     const bool plotFocusedWS = m_view->focusedOutWorkspace();
-    if (plotFocusedWS == true) {
+    int plotType = m_view->currentPlotType();
+    g_log.debug() << "...Plot Type return: " << plotType << std::endl;
+
+    // Qt comboBox returns absurd number for index of 0
+    if (plotType > 3) {
```

Comment:
This was a hack to get it working but should not survive here unless absolutely needed.